apps/system/js/utility_tray.js

```diff
@@ -236,7 +259,15 @@ window.UtilityTray = {
           evt.preventDefault();
         }
 
+        var oldDelta = this.lastDelta;
+
         this.onTouchMove(evt.touches[0]);
+
+        var move = this.lastDelta - oldDelta;
+        if (Math.abs(move) > 0) {
+          this.lastTime = evt.timeStamp;
+          this.lastMove = move;
+        }
```

Comment:
Ahhh, sorry I had misunderstood what `this.lastDelta` was. It's not the delta from the last touch move, but the delta from touch start. So you can ignore that second part from the comment above.